platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -1377,19 +1377,15 @@ public boolean isDetached() {
      */
     @Override
     public TeacherTreeView<Long> buildTeacherView(final ReconnectConfig reconnectConfig) {
-        switch (config.reconnectMode()) {
-            case "pullTopToBottom":
-            case "pullTwoPhasePessimistic":
-                return new TeacherPullVirtualTreeView<>(
-                        getStaticThreadManager(), reconnectConfig, this, state, pipeline);
-            default:
-                logger.warn(RECONNECT.getMarker(), "Unknown reconnect mode: " + config.reconnectMode());
-                logger.warn(RECONNECT.getMarker(), "Using default reconnect mode: push");
-                // fallthrough
-            case "push":
-                return new TeacherPushVirtualTreeView<>(
-                        getStaticThreadManager(), reconnectConfig, this, state, pipeline);
-        }
+        return switch (config.reconnectMode()) {
+            case "push" -> new TeacherPushVirtualTreeView<>(
```

Comment:
I suggest to make those constants public and use them here instead of replicating their values.

Reply:
OK